spec/simplecov_merger.rb

```diff
@@ -4,7 +4,7 @@
 
 module SimpleCov
   module ResultMerger
-    RSPEC_FILENAME_REGEX = %r{^(?<filename>[_\w\d/.]+\.rb)}
+    RSPEC_FILENAME_REGEX = %r{^(?<filename>[-_\w\d/.]+(?<extension>\.rb)?)}
```

Comment:
changes:
 - the source file, `bin/query-cloudwatch` does not have an `.rb` extension, so made the `.rb` optional
 - `bin/query-cloudwatch` also has dashes instead of underscores, so added `-` to the character class